app.js

```diff
@@ -66,9 +82,13 @@ function displayFriends() {
 }
 
 function displayMushrooms() {
-    // clear out the mushroom div
+    mushroomsEl.textContent = '';
+
 
     for (let i = 0; i < mushroomCount; i++) {
+        const mushroomCount = renderMushroom(mushroomsEl);
```

Comment:
If you look at the `renderMushroom()` function definition you'll notice it doesn't take any parameters. No need to pass in an argument here. 

Also - using `mushroomCount` for the variable name is confusing since that is already used as a variable in state. This variable you declare here should be named after the value it will hold, which is a DOM element that is returned from `renderMushroom()`. A better name would be `mushroomEl` or something like that.